rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,39 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+linter for the programming language the repo is written in, where one is
```

Comment:
If you're in Rails, you want to be linting Ruby and JS, so I've tweaked the wording a little.

```suggestion
linter for the major programming languages the repo is written in, where
```